src/main/java/io/minimum/minecraft/superbvote/configuration/SuperbVoteConfiguration.java

```diff
@@ -182,9 +182,10 @@ public VoteStorage initializeVoteStorage() throws IOException {
                 String database = configuration.getString("storage.mysql.database", "superbvote");
                 String table = configuration.getString("storage.mysql.table", "superbvote");
                 boolean readOnly = configuration.getBoolean("storage.mysql.read-only");
+                boolean ssl = configuration.getBoolean("storage.mysql.ssl");
```

Comment:
I think here, the default value should be added as 'true' otherwise users without the config file updated, will not use SSL by default.